test/modm/io/io_stream_test.cpp

```diff
@@ -624,6 +625,7 @@ IoStreamTest::testPrintf3()
 	(*stream).printf("%lld", longlong);
 	TEST_ASSERT_EQUALS_ARRAY("-9223372036854775806", device.buffer, 20);
 	(*stream).flush();
+#endif
```

Comment:
This was supposed to work by defining `__USE_MINGW_ANSI_STDIO`, but you're not supposed to define it directly, instead use something like `_POSIX_C_SOURCE`, but neither ones worked. Meh.